action/recipes/stitching_test.go

```diff
@@ -141,6 +141,12 @@ type makeFile struct {
 }
 
 func (base makeFile) MakeMe() error {
+	log.Printf("Path:       %s", base.Path)
```

Comment:
I'm not fully in the loop in regards of logging. Did we switch to slog or is log still the way to go?

Reply:
Normally we use `slog`, but in tests we use `log`.

Reply:
Totally overlooked that this is a test file. It makes no big difference but maybe we should use `t.Log()` but that would require this function to also take `*testing.T` as an input parameter and might be out of scope for this PR.

Reply:
`t.Log()` might be better than `slog`. I will put this into my todo list with low priority.